src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/ContainerBuilderExtensions.cs

```diff
@@ -177,8 +178,8 @@ private static void RegisterDefaultResourceProviders(this ContainerBuilder build
         private static void RegisterV2ResourceProviders(this ContainerBuilder builder)
         {
             builder.RegisterResourceProvider<NonhijackableV2HttpHandlerResourceProvider>(FeedType.HttpV2);
-            builder.RegisterResourceProvider<PackageTimestampMetadataResourceV2Provider>(FeedType.HttpV2);
-            builder.RegisterResourceProvider<PackageRegistrationMetadataResourceV2FeedProvider>(FeedType.HttpV2);
+            builder.RegisterResourceProvider<PackageTimestampMetadataResourceDatabaseProvider>(FeedType.HttpV2);
+            builder.RegisterResourceProvider<PackageRegistrationMetadataResourceDatabaseFeedProvider>(FeedType.HttpV2);
```

Comment:
good call

Reply:
My bad, resolved this accidentally.

Yes, `FeedType.HttpV2` is necessary. We are using client's APIs to fetch information to V2 and V3, and have created a dictionary of `FeedType` to client DI instance. So, `FeedType.HttpV3` maps to a client DI instance set up to download from V3, and `FeedType.HttpV2` maps to a client DI instance set up to query V2.

I don't think we actually use any of the client APIs for V2 anymore now that we query the gallery, so we _could_ replace all these DI dictionary shenanigans, but I'd rather not because it's not necessary. When I responded earlier I was thinking that I could replace the name `FeedType.HttpV2` to something like `FeedType.Database` or something to make things clearer, but I had forgotten that `FeedType` is defined in `NuGet.Protocol` and not in this project.

Reply:
Gotcha! 👍